### ISSUE TITLE
MozFest Style Guide -  Deadline July 19

### ISSUE THREAD

**Issue by gideonthomas** [MozillaFoundation/mozfest-design-2019#1]
### Description

Create a style guide for MozFest's visual direction for 2019. Document an extended version and a simplified version that can be easily shared with wranglers, volunteers etc.

The branding should draw from existing branding work standards happening for MoFo-wide content. The style should include elements of the 10th anniversary of MozFest and can optionally include elements of better machine decision making. More importantly, let's keep it simple so that it's achievable within a few weeks with ongoing touch-ups as needed.

#### Components

- [x] Moodboards
- [x] Color palette https://foundation.mozilla.org/en/docs/brand/design/color/
- [x] Typography https://foundation.mozilla.org/en/docs/brand/design/Typography/ 
- [x] Two new spaces icons
- [x] Icons https://foundation.mozilla.org/en/docs/brand/design/iconography/
- [x] Logos https://foundation.mozilla.org/en/docs/brand/design/logos/
- [x] Art textures
- [x] Master .ai file
- [ ] Add file to [wagtail page](https://foundation.mozilla.org/en/docs/brand/design/)
### Reference from last year

**GitHub ticket(s):** 
- [Moodboards](https://github.com/MozillaFoundation/mozfest-design-2018/issues/5)
- [Style guide](https://github.com/MozillaFoundation/mozfest-design-2018/issues/31)
- [Art textures](https://github.com/MozillaFoundation/mozfest-design-2018/issues/29)
- [Color](https://github.com/MozillaFoundation/mozfest-design-2018/issues/27)
- [Typography](https://github.com/MozillaFoundation/mozfest-design-2018/issues/28)

**Comment by gideonthomas** [comment]
@sabrinang how are we doing with this ticket?

**Comment by kristinashu** [comment]
@sabrinang I updated the components list above. Can you please add anything you think is missing and checkoff what is finished?

**Comment by sabrinang** [comment]
Style guide folder here:
https://drive.google.com/open?id=1iIsErPVIdjaYGJRCtjpJ-lor2KKgDYBz

It so far contains exported and curated color, iconography (includes all updated space icons), typography and lockups assets. 

Art textures, pattern tiles and moodboards are WIP.

**Comment by sabrinang** [comment]
Updated the .ai file in WIP folder (includes moodboards and art textures/elements): https://drive.google.com/open?id=1LC7oFlKxC6CNzBnSpBKTVVWt1GwYlik5&authuser=sabrina@mozillafoundation.org&usp=drive_fs

Redpen here: https://redpen.io/jkeb668d4aaac32ffe

I'm going to collect some feedback on this refined version on Monday then make simplified/extended documentation that is shareable. Assets haven't been made yet so there isn't many examples yet so it'll be included in doc/slides as they get created.

**Comment by sabrinang** [comment]
/cc @natalieworth 
I also updated MozFest stuff that could inform deck https://github.com/MozillaFoundation/Design/issues/373

**Comment by sabrinang** [comment]
Updating style guide extended deck here: https://docs.google.com/presentation/d/1TUAe-GJ2ICSRV4VeC_Zur0Fmc3oXY5GJm1U6RV7NIa0/edit?usp=sharing

**Comment by sabrinang** [comment]
Style guide folder in gdrive: https://drive.google.com/open?id=1iIsErPVIdjaYGJRCtjpJ-lor2KKgDYBz

Style guide deck (extended - internal reference): https://docs.google.com/presentation/d/1TUAe-GJ2ICSRV4VeC_Zur0Fmc3oXY5GJm1U6RV7NIa0/edit?usp=sharing
Style guide deck (simplified - shareable): https://docs.google.com/presentation/d/16QB9UBT218okSZdBhNtM1qQz3mcCrsGzIYlh73wa-9A/edit?usp=sharing

**Comment by kristinashu** [comment]
It's beautiful Sabs!!!

**Comment by sabrinang** [comment]
I've added the simplified gslide link on a wagtail page https://foundation.mozilla.org/en/docs/brand/design/reference/  The extended guide is for designers/contractors so that will be shared internally only. The elements will be incorporated to the Foundation style guide via other issues https://github.com/MozillaFoundation/Design/issues/385 and https://github.com/MozillaFoundation/Design/issues/388 which will show up in other forms of wagtail documentation too.

Closing this issue - asset/application examples will be added to the extended deck as we go and if new elements are generated they can be incorporated into the folder.
